Standardise non-regex patterns in query blocking upon loading of config (#12102)
grafana/mimir: docs/sources/mimir/configure/configure-blocked-queries.md

## Format queries to block

> Queries received by Mimir are parsed into PromQL expressions before blocking is applied. When `regex` is

Author: Good point, added that back.

## Format queries to block

> You can ignore this section if you are not using regex. If you are, you'll need to ensure that the regex
> will match against formatted queries as follows.

Reviewer: ```suggestion
You can ignore this section if you're not using a regular expression. If you are, you need to ensure that it matches against formatted queries as follows.
```

Use Mimirtool's `mimirtool promql format <query>` command to apply the Prometheus formatter to a query
for use in a blocked query pattern.

> Queries received by Mimir are parsed into PromQL expressions before blocking is applied. When regex is

Reviewer: ```suggestion
Mimir parses queries into PromQL expressions before blocking is applied. When a regular expression is
```

not used, the pattern from the blocked queries is also similarly parsed before being compared against

> the formatted representation of the parsed query. When regex is used, the pattern is used as-is. This

Reviewer: ```suggestion
the formatted representation of the parsed query. When a regular expression is used, the pattern is used as-is. This
```
